packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -101,18 +127,31 @@ test.describe('severity-1 #smoke', () => {
       params.delete('redirect_uri');
 
       // reload email-first page without redirect_uri, but with React experiment params
-      await signupReact.goToEmailFirstAndCreateAccount(params, email, PASSWORD);
+      await signupReact.goto('/', params);
+
+      await signupReact.fillOutEmailForm({ email: email, submit: true });
+
+      await signupReact.fillOutSignupForm({
+        password: PASSWORD,
+        age: AGE_21,
+        submit: true,
+      });
 
+      await signupReact.fillOutCodeForm({ email: email, submit: true });
       // redirectUri should have fallen back to the clientInfo config redirect URI
       // Expect to be redirected to relier
       await page.waitForURL(target.relierUrl);
+
       expect(await relier.isLoggedIn()).toBe(true);
+
       await relier.signOut();
     });
 
     test('signup oauth webchannel - sync mobile or FF desktop 123+', async ({
-      syncBrowserPages: { page, signupReact, login },
+      syncBrowserPages: { page, login, signupReact },
+      email,
     }) => {
+      test.fixme(true, 'Fix required as of 2024/03/18 (see FXA-9306).');
```

Comment:
We also had FXA-9096 filed specifically for this test, which was recently merged. Is it still flaky?

Reply:
I am still experiencing some flake locally with these tests unfortunately. I added this task out of an abundance of caution because I'd like to dig into it more. Is it covering a very critical path? Is it better to let it flake occasionally than to have the fixme?

Reply:
👍 Just wanted to double check it's still flaking.

Firefox has been working on SYNC-3768 and this is part of the FxA side work. Looks like FXA-9100 blocks that epic from being finished - ideally we should look at this in the next month or so or shortly after, preferably before this feature makes it to ESR. I think it's OK to disable them in the meantime.